bin/main.rb

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
-require 'timeout'
+
+require '../lib/game_methods'
```

Comment:
**[OPTIONAL]** 
* Let's use `require_relative` as opposed to `require` since it's the ruby convention of importing files.